### PR TITLE
Abandon in-file translations, document full-file translation guidance

### LINKED ISSUE #229: Abandon in-file translations, document full-file translation guidance
The [current schema](http://ocds.open-contracting.org/standard/r/1__0__0/en/schema/reference/#language) uses an approach to indicate language based on suffixes to property names. 

``` json
{
    "language": "en",
    "tender": {
        "item": {
                "description":"Software consultancy services",
                "description_es":"Servicios de consultoria en software",
                "description_fr":"Services de conseil en logiciels"
        }
    }
}
```

This was discussed in #21 during the beta, and was chosen on the understanding that we wanted to avoid depth where we could (for easier flat renderings of the the data). However, on reflection, the approach we have adopted may not have been the most appropriate.

In particular, @elf-pavlik has pointed out [here](https://github.com/open-contracting/standard/issues/40#issuecomment-134295223) in #40 that [JSON-LD](http://www.w3.org/TR/json-ld/#string-internationalization) uses language maps, and it would be much easier to create a JSON LD rendering of our data this way. 

``` json
{
    "@context": { "@language": "en" },
    "tender": {
        "item": {
              "description": {
                "en" :"Software consultancy services",
                "es": "Servicios de consultoria en software",
                "fr": "Services de conseil en logiciels"
        }
    }
}
```

Our flattening approach would also just render this to 'description/en', 'description/es' and so-on, which seems fairly intuitive, and our earlier fears about over-use of objects does not appear a major one.

However, to change to language maps would potentially be a backwards incompatible update. 

It might make sense to look at doing it early: but would definitely need wide discussion.

### PR DESCRIPTION
**Pull request by duncandewhurst**
* Closes #229

### CODE REVIEW

**Review by odscjen — APPROVED** [review]

File: docs/guidance/map.md [hunk 1]
```diff
@@ -110,6 +110,7 @@ map/organization_personal_identifiers
 map/organizational_units
 map/organization_classifications
 map/beneficial_ownership
+map/translations
```
Comment (jpmckinney) [inline docs/guidance/map.md]: I think we instead want a paragraph with a link to this page under "Localize OCDS to your context".

File: docs/schema/reference.md [hunk 2]
```diff
@@ -34,35 +34,11 @@ There can be cases where a publisher needs to remove, rather than update, a valu
...
-
-A contract for ‘Software consultancy services’ is published in a release with the default language sent to ‘en’ (the ISO639-1 code for English). The following examples give the description of an item as English, French and Spanish.
-
-**json**
-
-```{jsoninclude} ../examples/language_localization/language.json
-:jsonpointer:
-:expand: tender,item
+```{deprecated} 1.2
+  Use [full-file translations](../guidance/map/translations.md) instead.
```
Comment (jpmckinney) [inline docs/schema/reference.md]: This entire section is deprecated, so I think instead delete the "Many publishers..." paragraph and:
```suggestion
OCDS 1.1 allowed data to be published in multiple languages by suffixing a language code to a field name: for example, `title` for the default language and `title_es` for Spanish. OCDS 1.2 uses [full-file translations](../guidance/map/translations), instead.
```

**Review by jpmckinney — COMMENTED** [review]
I don't think this closes #1064, which wants to cover translations of extensions.

I'll make translations.md copy-edits directly.

We can delete examples/language_localization/language.csv and its language.json

----

I updated manage.py to make these nontranslatable: postalCode, email, telephone, faxNumber, milestone code. These have publisher-specific formats or codelists, or are missing a validation property like `"format": "email"` #881.

Comparing `i18n.csv` to the patternProperties, we've removed deprecated fields (good) and added:

* These 3 address components can indeed be translated
  * Address.locality
  * Address.region
  * Address.streetAddress
* I suppose we just missed these 3 originally:
  * Amendment.description
  * RelatedProcess.title
  * Tender.procurementMethodDetails
* These fields are new:
  * Award.statusDetails
  * Contract.statusDetails
  * Tender.statusDetails
  * Tender.exclusionGrounds

Also, in OCDS 1.0, `Organization.name` was translatable. In OCDS 1.1, `OrganizationReference.name` was not translatable. In OCDS 1.2, both are translatable. In other words, both `buyer/name` and the `.name` of the corresponding entry in the `parties` array are translatable. This is good.

**Comment by jpmckinney** [comment]
I've pushed my edits to the new guidance page, for review.

File: docs/schema/reference.md (hunk 2 above)
Comment (duncandewhurst) [reply docs/schema/reference.md]: Paragraph removed in cb37d9935cab44a24c293647f520554ad6a2ee6d.

File: docs/guidance/map.md (hunk 1 above)
Comment (duncandewhurst) [reply docs/guidance/map.md]: Done in e88f438da8c611674d9cb7a448950d2aa4db6776

**Comment by duncandewhurst** [comment]
> We can delete examples/language_localization/language.csv and its language.json

Done in 37a024addef69ca648d720a0433f1ba68f524ead

The revised guidance page and the changes to `manage.py` look good to me.

**Review by jpmckinney — APPROVED** [review]

**Pull request merged**